packages/core/src/test/shared/utilities/pathFind.test.ts

```diff
@@ -4,19 +4,23 @@
  */
 
 import assert from 'assert'
+import sinon from 'sinon'
 import * as vscode from 'vscode'
 import * as os from 'os'
 import * as path from 'path'
-
 import * as testutil from '../../testUtil'
-import { findTypescriptCompiler, getVscodeCliPath } from '../../../shared/utilities/pathFind'
 import { fs } from '../../../shared'
+import { findSshPath, findTypescriptCompiler, getVscodeCliPath } from '../../../shared/utilities/pathFind'
+import * as programUtils from '../../../shared/utilities/programUtils'
+
+function isWin(): boolean {
+    return process.platform === 'win32'
```

Comment:
i guess we could add this to `packages/core/src/shared/utilities/osUtils.ts` , since it's somewhat common.

Reply:
I found a function in `src/shared/vscode/env.ts` that actually did the same thing, so I removed it.